src/overlays/actors/ovl_Obj_Lift/z_obj_lift.c

```diff
@@ -14,11 +14,13 @@ void ObjLift_Init(Actor* thisx, GlobalContext* globalCtx);
 void ObjLift_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ObjLift_Update(Actor* thisx, GlobalContext* globalCtx);
 void ObjLift_Draw(Actor* thisx, GlobalContext* globalCtx);
+void func_80B9651C(ObjLift* this);
+void func_80B96560(ObjLift* this, GlobalContext* globalCtx);
+void func_80B9664C(ObjLift* this);
+void func_80B967C0(ObjLift* this);
+void func_80B96678(ObjLift* this, GlobalContext* globalCtx);
+void func_80B96840(ObjLift* this, GlobalContext* globalCtx);
```

Comment:
just an organizational thing. we usually break after the first 4 main functions, and i grouped the "setup" functions and the action functions separately 
```suggestion
void ObjLift_Destroy(Actor* thisx, GlobalContext* globalCtx);
void ObjLift_Update(Actor* thisx, GlobalContext* globalCtx);
void ObjLift_Draw(Actor* thisx, GlobalContext* globalCtx);

void func_80B9651C(ObjLift* this);
void func_80B9664C(ObjLift* this);
void func_80B967C0(ObjLift* this);

void func_80B96560(ObjLift* this, GlobalContext* globalCtx);
void func_80B96678(ObjLift* this, GlobalContext* globalCtx);
void func_80B96840(ObjLift* this, GlobalContext* globalCtx);
```